utils/repeatUntil.ts

```diff
@@ -0,0 +1,26 @@
+import { TaskEither } from "fp-ts/lib/TaskEither";
```

Comment:
consider using `withRetries` instead: https://github.com/pagopa/io-ts-commons/blob/master/src/tasks.ts#L46
with some small delay (the probability to have colliding codes is very low)